### PR TITLE
packages/prometheus/docs:  Replace `darasteam` with `datastream`

### PR DESCRIPTION
**Pull request by limistah**
## What does this PR do?
The page https://docs.elastic.co/integrations/prometheus#dashboard states:

> Prometheus integration is shipped including default overview dashboard. Default dashboard works only for remote_write datastream and collector `darastream`, if metrics are scraped from the Prometheus server metrics endpoint.

The `darastream` should be `datastream` and read like

> Prometheus integration is shipped including default overview dashboard. Default dashboard works only for remote_write datastream and collector `datastream`, if metrics are scraped from the Prometheus server metrics endpoint.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Screenshots
image

### CODE REVIEW

**Comment by limistah** [comment]
`/test`

**Review by constanca-m — APPROVED** [review]

**Review by gizas — APPROVED** [review]

**Comment by limistah** [comment]
The CI keeps failing, this is a typo fix. I'm not sure if I am doing anything wrong to cause the CI to fail. Can someone help?

**Comment by limistah** [comment]
👍🏾

**Comment by limistah** [comment]
Hi @tetianakravchenko 

Your review is pending on this PR

Thanks

**Comment by limistah** [comment]
👍🏾

**Review by tetianakravchenko — APPROVED** [review]

**Comment by mrodm** [comment]
Hi @limistah, please update your branch with the latest contents from main branch. There was an important PR merged updating the CI pipelines. Thanks!

**Comment by limistah** [comment]
Hi @mrodm I have updated the PR

Thank you for the info

**Comment by limistah** [comment]
👍

**Review by shmsr — APPROVED** [review]

File: packages/prometheus/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.0"
+  changes:
+    - description: Typo Fix darastream instead of datastream
```
Comment (shmsr) [inline packages/prometheus/changelog.yml:4]:
```suggestion
    - description: Fix typo: Replace "darastream" with "datastream"
```

**Review by shmsr — CHANGES_REQUESTED** [review]
Left one comment.

**Review by shmsr — APPROVED** [review]

**Comment by shmsr** [comment]
@limistah Thanks for your contribution!

File: packages/prometheus/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.0"
+  changes:
+    - description: 'Fix typo: Replace "darastream" with "datastream"'
```
Comment (shmsr) [inline packages/prometheus/changelog.yml:4]:
```suggestion
    - description: Fix typo: Replace "darastream" with "datastream"
```

Comment (shmsr) [reply packages/prometheus/changelog.yml:4]: remove single quotes

**Review by shmsr — APPROVED** [review]
Thanks for your contribution @limistah!

**Comment by aliabbas-elastic** [comment]
@limistah Can you resolve the conflicts here?

File: packages/prometheus/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.0"
```
Comment (aliabbas-elastic) [inline packages/prometheus/changelog.yml:2]:
```suggestion
- version: "1.14.1"
```
As this is a minor fix in documentation bumping the patch version would suffice.

**Review by aliabbas-elastic — CHANGES_REQUESTED** [review]

**Comment by ishleenk17** [comment]
@limistah : Please fix the comments. We can then retrigger the build and commit the change

**Comment by ishleenk17** [comment]
@limistah : Lets take this PR forward!

**Comment by aliabbas-elastic** [comment]
@limistah Please also bump up the `manifest.yml` version as well

**Comment by limistah** [comment]
> @limistah Please also bump up the `manifest.yml` version as well

Done with this commit https://github.com/elastic/integrations/pull/7914/commits/70c889566ac399b1d7920205c9478592e72dc4b4

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

File: packages/prometheus/docs/README.md [hunk 4]
```diff
@@ -261,6 +261,7 @@ spec:
   sessionAffinity: None
   type: ClusterIP
 ```
+
```
Comment (aliabbas-elastic) [inline packages/prometheus/docs/README.md:264]: Maybe this is causing the build failure. Can you run once `elastic-package build` and `elastic-package check` once?

Comment (limistah) [reply packages/prometheus/docs/README.md]: There was a marshal error with the changelog.yml file. I have fixed this.

image

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

**Pull request merged**